Wrappers/Python/cil/processors/RingRemover.py

```diff
@@ -26,30 +26,29 @@ class RingRemover(Processor):
     
     '''
         RingRemover Processor: Removes vertical stripes from a DataContainer(ImageData/AcquisitionData) 
-        the algorithm in https://doi.org/10.1364/OE.17.008567
+        using the algorithm in https://doi.org/10.1364/OE.17.008567
 
-    '''
-        
-    def __init__(self, decNum, wname, sigma, info = True):
-        
-        '''
-    
         Parameters
         ----------
-        decNum : number of wavelet decompositions
+        decNum : int
+            Number of wavelet decompositions
         
-        wname : (str) name of wavelet filter from pywt 
-            Example: 'db1' -- 'db35', 'haar'
+        wname : str
+            Name of wavelet filter from pywt e.g. 'db1' -- 'db35', 'haar'
     
-        sigma : Damping parameter in Fourier space.
+        sigma : float 
+            Damping parameter in Fourier space
```

Comment:
Can you add the effect of changing this parameter? E.g. larger sigma = more damping = some effect for the ring remover